dali/pipeline/pipeline.cc

```diff
@@ -120,9 +120,23 @@ void Pipeline::AddOperator(OpSpec spec, const std::string& inst_name) {
   //               - make it as an arg of nvJPEGDecoderNew
 
   // nvJGPEGDecoder operator require a special case to be divided in two stages (CPU and Mixed-GPU)
-  const bool nvjpeg_splitting = spec.name() == "nvJPEGDecoderSplitted";
-  if (nvjpeg_splitting) {
-    AddSplittedNvJpegDecoder(spec, inst_name);
+  std::string fullname = spec.name();
```

Comment:
note: There is a plan to remove all those "Splitted" operators and use an option in nvJPEGDecoder: https://jirasw.nvidia.com/browse/DALI-603
In that task I'll refactor this code to something more generic